.github/CODEOWNERS

```diff
@@ -285,17 +285,24 @@
 
 # The following is managed by `inv lint-components` -- DO NOT EDIT
 # BEGIN COMPONENTS
-/comp @DataDog/agent-shared-components
-/comp/agent @DataDog/agent-shared-components
+/comp @DataDog/agent-runtimes
+/comp/agent @DataDog/agent-runtimes
 /comp/aggregator @DataDog/agent-metrics-logs
-/comp/api @DataDog/agent-shared-components
+/comp/api @DataDog/agent-configuration
 /comp/checks @DataDog/agent-metrics-logs
 /comp/collector @DataDog/agent-metrics-logs
-/comp/core @DataDog/agent-shared-components
```

Comment:
I believe for the components you need to update the codeowner comment in the comp definition